etc/taskcluster/macos/README.md

```diff
@@ -15,10 +15,20 @@ cd etc/taskcluster/macos
 ./salt-ssh '*' state.apply test=True
 ```
 
-## Servers
+## (Re)deploying a server
 
-Servers are provisioned manually from MacStadium.
-The `config/roster` file lists them by DNS name.
+* Place an order or file a ticket with MacStadium to get a new hardware or reinstall an OS.
+
+* Change the administrator password to one generated with
+  `</dev/urandom tr -d -c 'a-zA-Z' | head -c 8; echo`
+  (this short because of VNC),
+  and save it in the shared 1Password account.
+
+* Give the public IPv4 address a DNS name through Cloudflare.
+
+* Add a correponding entry in the `config/roster` file.
+
+* Log in through VNC, and run `xcode-select --install`
```

Comment:
https://github.com/servo/servo/wiki/SaltStack-Administration has a number of other steps we follow right now.

Reply:
I’m hoping to make some of those taken care of by Salt or by tasks, but installing XCode requires GUI interaction.

Reply:
Also, this document is intended to expand over time as we do more, it is not at an end state. This is documenting what I’ve done so far and how to reproduce it.